event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinConversionException.java

```diff
@@ -11,7 +11,7 @@ public CristinConversionException(String message) {
     }
 
     public static CristinConversionException fromFailure(Failure<?> failure) {
-        var message = failure.getException().getMessage();
+        var message = failure.getException();
         return new CristinConversionException(String.format("%s %s", EXCEPTION_MESSAGE, message));
```

Comment:
Updating error report to contain whole exception, we need to know where exactly in code exception has occurred.